Sources/MockoloFramework/Templates/ClassTemplate.swift

```diff
@@ -94,22 +93,6 @@ private func extraInitsIfNeeded(_ entities: [(String, Model)],
         if declType == .protocolType {
             needBlankInit = !hasBlankInit
             needParamedInit = declaredInitParamsPerInit.isEmpty && !extraInitParamCandidates.isEmpty
-        } else {
-            var matchingParamsFound = false
-            for declaredParams in declaredInitParamsPerInit {
-                if declaredParams.count == extraInitParamCandidates.count {
-                    let declaredParamsByName = declaredParams.sorted(path: \.name)
-                    let matchingParams = zip(extraInitParamCandidatesSorted, declaredParamsByName).filter { $0.name == $1.name && $0.type.typeName == $1.type.typeName }
-                    if !matchingParams.isEmpty {
-                        matchingParamsFound = true
-                        break
-                    }
-                }
-            }
-            
-            if matchingParamsFound || extraInitParamCandidates.isEmpty {
-                needParamedInit = false
-            }
```

Comment:
How come we don't need any of this anymore? Admittedly I don't have context on the role that `needParamedInit` plays, so I'm assuming it's related to that

Reply:
extra init's are still generated for protocol mocks but not for class mocks.  this change handles that.